BUG: linking issues when including ewah_bool_utils at compilation time

Even after #16, another issue remains when trying to compile yt with ewah-bool-utils as a drop in replacement for the vendored equivalent:

<details> <summary> Log </summary>

```
Installing collected packages: yt
  Attempting uninstall: yt
    Found existing installation: yt 4.2.dev0
    Uninstalling yt-4.2.dev0:
      Successfully uninstalled yt-4.2.dev0
  Running setup.py develop for yt
    error: subprocess-exited-with-error
    
    × python setup.py develop did not run successfully.
    │ exit code: 1
    ╰─> [51 lines of output]
        Unable to compile OpenMP test program so Cython
        extensions will be compiled without parallel support
        running develop
        /Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        /Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        running egg_info
        writing yt.egg-info/PKG-INFO
        writing dependency_links to yt.egg-info/dependency_links.txt
        writing entry points to yt.egg-info/entry_points.txt
        writing requirements to yt.egg-info/requires.txt
        writing top-level names to yt.egg-info/top_level.txt
        reading manifest file 'yt.egg-info/SOURCES.txt'
        reading manifest template 'MANIFEST.in'
        warning: no previously-included files found matching 'scripts/pr_backport.py'
        warning: no previously-included files found matching 'yt/utilities/lib/cykdtree/c_kdtree.cpp'
        no previously-included directories found matching 'tests'
        no previously-included directories found matching 'docker'
        no previously-included directories found matching 'answer-store'
        warning: no files found matching 'README*' under directory 'yt'
        warning: no files found matching '*.svgz' under directory 'doc'
        warning: no files found matching '*.pdf' under directory 'doc'
        warning: no previously-included files found matching 'doc/cheatsheet.pdf'
        no previously-included directories found matching 'doc/source/reference/api/generated'
        no previously-included directories found matching 'doc/build'
        no previously-included directories found matching '.tours'
        warning: no files found matching '*.fragmentshader' under directory 'yt/visualization/volume_rendering/shaders'
        warning: no files found matching '*.vertexshader' under directory 'yt/visualization/volume_rendering/shaders'
        warning: no previously-included files found matching '.codecov.yml'
        warning: no previously-included files found matching '.coveragerc'
        warning: no previously-included files found matching '.git-blame-ignore-revs'
        warning: no previously-included files found matching '.gitmodules'
        warning: no previously-included files found matching '.hgchurn'
        warning: no previously-included files found matching '.mailmap'
        warning: no previously-included files found matching '.pre-commit-config.yaml'
        warning: no previously-included files found matching 'clean.sh'
        warning: no previously-included files found matching 'nose_answer.cfg'
        warning: no previously-included files found matching 'nose_unit.cfg'
        adding license file 'COPYING.txt'
        writing manifest file 'yt.egg-info/SOURCES.txt'
        running build_ext
        building 'yt.geometry.particle_oct_container' extension
        clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include -Iyt/utilities/lib/ -I/Users/robcleme/.pyenv/versions/yt-dev/include -I/Users/robcleme/.pyenv/versions/3.10.8/include/python3.10 -I/Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/numpy/core/include -c yt/geometry/particle_oct_container.cpp -o build/temp.macosx-13.0-x86_64-cpython-310/yt/geometry/particle_oct_container.o -std=c++14
        yt/geometry/particle_oct_container.cpp:809:10: fatal error: 'ewah.h' file not found
        #include "ewah.h"
                 ^~~~~~~~
        1 error generated.
        error: command '/usr/bin/clang' failed with exit code 1
        While building 'yt.geometry.particle_oct_container' following error was raised:
         command '/usr/bin/clang' failed with exit code 1
        [end of output]
    
    note: This error originates from a subprocess, and is likely not a problem with pip.
  Rolling back uninstall of yt
  Moving to /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/bin/yt
   from /private/var/folders/xh/yfmp7l9n3x50ln2yvrndt7c801pbkm/T/pip-uninstall-_nlau_0d/yt
  Moving to /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/lib/python3.10/site-packages/__editable__.yt-4.2.dev0.pth
   from /private/var/folders/xh/yfmp7l9n3x50ln2yvrndt7c801pbkm/T/pip-uninstall-7mwv85st/__editable__.yt-4.2.dev0.pth
  Moving to /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/lib/python3.10/site-packages/__editable___yt_4_2_dev0_finder.py
   from /private/var/folders/xh/yfmp7l9n3x50ln2yvrndt7c801pbkm/T/pip-uninstall-7mwv85st/__editable___yt_4_2_dev0_finder.py
  Moving to /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/lib/python3.10/site-packages/__pycache__/__editable___yt_4_2_dev0_finder.cpython-310.pyc
   from /private/var/folders/xh/yfmp7l9n3x50ln2yvrndt7c801pbkm/T/pip-uninstall-7mwv85st/__pycache__/__editable___yt_4_2_dev0_finder.cpython-310.pyc
  Moving to /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/lib/python3.10/site-packages/yt-4.2.dev0.dist-info/
   from /Users/robcleme/.pyenv/versions/3.10.8/envs/yt-dev/lib/python3.10/site-packages/~t-4.2.dev0.dist-info
error: subprocess-exited-with-error

× python setup.py develop did not run successfully.
│ exit code: 1
╰─> [51 lines of output]
    Unable to compile OpenMP test program so Cython
    extensions will be compiled without parallel support
    running develop
    /Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    /Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    running egg_info
    writing yt.egg-info/PKG-INFO
    writing dependency_links to yt.egg-info/dependency_links.txt
    writing entry points to yt.egg-info/entry_points.txt
    writing requirements to yt.egg-info/requires.txt
    writing top-level names to yt.egg-info/top_level.txt
    reading manifest file 'yt.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no previously-included files found matching 'scripts/pr_backport.py'
    warning: no previously-included files found matching 'yt/utilities/lib/cykdtree/c_kdtree.cpp'
    no previously-included directories found matching 'tests'
    no previously-included directories found matching 'docker'
    no previously-included directories found matching 'answer-store'
    warning: no files found matching 'README*' under directory 'yt'
    warning: no files found matching '*.svgz' under directory 'doc'
    warning: no files found matching '*.pdf' under directory 'doc'
    warning: no previously-included files found matching 'doc/cheatsheet.pdf'
    no previously-included directories found matching 'doc/source/reference/api/generated'
    no previously-included directories found matching 'doc/build'
    no previously-included directories found matching '.tours'
    warning: no files found matching '*.fragmentshader' under directory 'yt/visualization/volume_rendering/shaders'
    warning: no files found matching '*.vertexshader' under directory 'yt/visualization/volume_rendering/shaders'
    warning: no previously-included files found matching '.codecov.yml'
    warning: no previously-included files found matching '.coveragerc'
    warning: no previously-included files found matching '.git-blame-ignore-revs'
    warning: no previously-included files found matching '.gitmodules'
    warning: no previously-included files found matching '.hgchurn'
    warning: no previously-included files found matching '.mailmap'
    warning: no previously-included files found matching '.pre-commit-config.yaml'
    warning: no previously-included files found matching 'clean.sh'
    warning: no previously-included files found matching 'nose_answer.cfg'
    warning: no previously-included files found matching 'nose_unit.cfg'
    adding license file 'COPYING.txt'
    writing manifest file 'yt.egg-info/SOURCES.txt'
    running build_ext
    building 'yt.geometry.particle_oct_container' extension
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include -Iyt/utilities/lib/ -I/Users/robcleme/.pyenv/versions/yt-dev/include -I/Users/robcleme/.pyenv/versions/3.10.8/include/python3.10 -I/Users/robcleme/.pyenv/versions/yt-dev/lib/python3.10/site-packages/numpy/core/include -c yt/geometry/particle_oct_container.cpp -o build/temp.macosx-13.0-x86_64-cpython-310/yt/geometry/particle_oct_container.o -std=c++14
    yt/geometry/particle_oct_container.cpp:809:10: fatal error: 'ewah.h' file not found
    #include "ewah.h"
             ^~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit code 1
    While building 'yt.geometry.particle_oct_container' following error was raised:
     command '/usr/bin/clang' failed with exit code 1
    [end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
```

</details>

--- a/ewah_bool_utils/__init__.py
+++ b/ewah_bool_utils/__init__.py
@@ -2,3 +2,12 @@
 
 
 from .ewah_bool_wrap import *
+
+
+def get_include():
+    """
+    Returns the directory that contains ewah*.h headers
+    """
+    import os
+
+    return os.path.abspath(os.path.join(os.path.dirname(__file__), "cpp"))

